devel/sbt-0.13/Portfile

```
@@ -0,0 +1,69 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+PortGroup       java 1.0
+
+name            sbt-0.13
+version         0.13.18
+revision        0
+categories      devel java
+license         BSD
+maintainers     nomaintainer
+platforms       darwin
+
+description     Simple build tool (sbt) simplifies building Scala projects
+
+long_description \
+    Simple build tool (sbt) is provides unintrusive and easy to set up \
+    for simple Scala projects. All configuration, customization, and \
+    extension are done in Scala.  Sbt supports continuous compilation \
+    and testing with triggered execution in mixed Scala/Java projects.
+
+homepage        https://www.scala-sbt.org/
+master_sites    https://repo.typesafe.com/typesafe/ivy-releases/org.scala-sbt/sbt-launch/${version}
+distname        sbt-launch
+dist_subdir     ${name}/${version}
+
+checksums       md5     c660658ea9b1300d31aaa713f214b1b4 \
+                sha1    db44e7b60ed5c82ea00315ad223e3a8cc3d086e5 \
+                rmd160  a61cd251a1593dafa0d1452dbb615b4fb1fd5a19 \
```

There's probably no reason to list md5 or sha1 checksums anymore.
```suggestion
checksums       rmd160  a61cd251a1593dafa0d1452dbb615b4fb1fd5a19 \
```